fetchmail_attach_from_folder/model/fetchmail_server.py

```diff
@@ -253,7 +253,7 @@ def fields_view_get(self, cr, user, view_id=None, view_type='form',
                 if field.tag == 'field' and field.get('name') in modifiers:
                     field.set('modifiers', simplejson.dumps(
                         dict(
-                            eval(field.attrib['modifiers'],
+                            safe_eval(field.attrib['modifiers'],
```

Comment:
@gurneyalex it is strange that the previous build did not notice it, it was all green https://travis-ci.org/OCA/server-tools/builds/109303252